src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -77,6 +77,7 @@ function isContextProvider(fiber : Fiber) : boolean {
 exports.isContextProvider = isContextProvider;
 
 function popContextProvider() : void {
+  invariant(index >= 0, 'Unexpected context pop');
```

Comment:
I'd prefer `> -1` here because we already compare to `-1` in a few parts of the file.